src/experimental/safe_html_transform/configure.zcml

```diff
@@ -17,6 +17,15 @@
         provides="Products.GenericSetup.interfaces.EXTENSION"
         />
 
+    <genericsetup:registerProfile
+      name="uninstall"
+      title="experimental.safe_html_transform uninstall"
+      directory="profiles/uninstall"
+      description="Uninstalls the experimental.safe_html_transform package"
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      />
+
+    
```

Comment:
@tisto @lentinj  Can you provide a sample example of add-on where uninstall profile is used ?